docs/get-started/dev-big-picture.qmd

```diff
@@ -0,0 +1,150 @@
+---
+title: "The big picture: Builder"
+jupyter:
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+While the "Basic Use" section covered how to configure and build a site with quartodoc, this section focuses on using quartodoc as a python program.
+
+Programming with quartodoc will help with debugging, tinkering, and extending things.
```

Comment:
It might be helpful to users to explicitly say this section is not necessary to build a basic site, rather, is for developer/debugging purposes?